src/dsp_tools/utils/xml_upload_stash.py

```diff
@@ -65,12 +62,34 @@ def _log_unable_to_upload_xml_resource(
     logger.warning(err_msg, exc_info=True)
 
 
+def _log_iri_does_not_exist_error(
```

Comment:
`log_unable_to_retrieve_resource` should be private as well (with _ prefixed), just to be consistent.

Reply:
This function is not private because it is called up by the xmlupload.py script. I put it in this script because this is a "helper" script for that function during the code that uploads the stashed links.